libobs/util/windows/window-helpers.c

```diff
@@ -469,6 +469,13 @@ static int window_rating(HWND window, enum window_priority priority,
 	} else if (priority == WINDOW_PRIORITY_TITLE) {
 		val = title_val == 0 ? 0 : 0x7FFFFFFF;
 
+	} else if (priority == WINDOW_PRIORITY_TITLE_START) {
+		int count = 0;
+		while (title[count] && cur_title.array[count] &&
+		       title[count] == cur_title.array[count]) {
+			count++;
+		}
```

Comment:
```suggestion
		int count = 0;
		size_t title_len = strlen(title);
		size_t max_len = min(title_len, cur_title.len);
		while (count < max_len && title[count] &&
		       cur_title.array[count] &&
		       title[count] == cur_title.array[count]) {
			count++;
		}
```

I tested it locally, and the current code can crash. Try this code instead.